README.md

```diff
@@ -3,7 +3,7 @@
 
 [![osrm-backend CI](https://github.com/Project-OSRM/osrm-backend/actions/workflows/osrm-backend.yml/badge.svg)](https://github.com/Project-OSRM/osrm-backend/actions/workflows/osrm-backend.yml) [![Codecov](https://codecov.io/gh/Project-OSRM/osrm-backend/branch/master/graph/badge.svg)](https://codecov.io/gh/Project-OSRM/osrm-backend) [![Discord](https://img.shields.io/discord/1034487840219860992)](https://discord.gg/es9CdcCXcb)
 
-High performance routing engine written in C++14 designed to run on OpenStreetMap data.
```

Comment:
Wanted to make it `modern C++`, but then realised that C++17 is probably not very modern taking into account that C++23 is going to be released next year :)

Reply:
Btw if we are going to bump compiler versions probably nothing prevents from using C++20(not sure what can be really useful there though, but newer C++ standard is always better than older)